scripts/Makefile.connectal.build

```diff
@@ -91,7 +93,8 @@ BSCPATHFLAGS=  -bdir $(DTOP)/obj -vdir $(DTOP)/verilog -simdir $(DTOP)/obj -info
 BSCFLAGS_COMMON += $(BSCWARNFLAGS) $(BSCOPTFLAGS) $(BSCPATHFLAGS) --wait-for-license
 PROF_FLAGS= #-pg
 export SIM_CFLAGS= $(CFLAGS_PROJECT) -fPIC -Ijni -I $(CONNECTALDIR)/cpp -I $(CONNECTALDIR) -D SIMULATION $(BSVDEFINES) -O $(PROF_FLAGS)
-export SIM_CXXFLAGS= $(CXXFLAGS_PROJECT) -fPIC -Ijni -I $(CONNECTALDIR)/cpp -I $(CONNECTALDIR) -D SIMULATION $(BSVDEFINES) -O $(PROF_FLAGS)
+export BSIMV_CXXFLAGS= $(CXXFLAGS_PROJECT) -I $(CONNECTALDIR)/cpp -I $(CONNECTALDIR) -D SIMULATION $(BSVDEFINES) -O $(PROF_FLAGS)
+export SIM_CXXFLAGS= -fPIC -Ijni $(BSIMV_CXXFLAGS)
```

Comment:
but why are SIM_CXXFLAGS being passed to bsc? Are you using bsc to compile for another simulator?